pom.xml

```diff
@@ -387,8 +433,7 @@
               <detectJavaApiLink>false</detectJavaApiLink>
               <detectLinks>false</detectLinks>
               <detectOfflineLinks>false</detectOfflineLinks>
-              <doclint>none</doclint>
-              <source>8</source>
```

Comment:
source needed I think

Reply:
Javadoc plugin will use _maven.compiler.source_, see: https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html

Reply:
right but same than https://github.com/apache/johnzon/pull/96#discussion_r1021642715 (in particular using a parent which is not only ours like apache parent which sometimes make other choices than the defaults)